backend/hubs/serializers/hub.py

```diff
@@ -68,6 +72,35 @@ def get_location(self, obj):
             return LocationSerializer(obj.location.all(), many=True).data
         return None
 
+    def get_local_ambassador(self, obj):
+            localAmbassador = LocalAmbassador.objects.filter(hub_id=obj.id)
+            if localAmbassador.exists():
+                return LocalAmbassadorSerializer(localAmbassador[0]).data
+            return None
+
+class LocalAmbassadorSerializer(serializers.ModelSerializer):
+    title = serializers.SerializerMethodField()
+    user = serializers.SerializerMethodField()
+    title_de = serializers.SerializerMethodField()
```

Comment:
Instead of returning both `title` and `title_de` I'd detect the language of the user here and just return the title attribute in the correct language similar to here: https://github.com/climateconnect/climateconnect/blob/bc7269b9551771e8ae95ff7caec8009bd112e512/backend/climateconnect_api/serializers/badge.py#L19-L20